src/ActionsImporter/Commands/Configure.cs

```diff
@@ -13,6 +13,7 @@ protected override Command GenerateCommand(App app)
         ArgumentNullException.ThrowIfNull(app);
 
         var command = base.GenerateCommand(app);
+        command.AppendPrereleaseOption();
```

Comment:
```suggestion
```
Configure doesn't need to know about the prerelease option, right?

Reply:
Imo it's worth configuring for all of the options for consistency. If every other option implements the `--prerelease` flag, it would feel surprising to an end user if the flag wasn't implemented in configure.

I'm also imagining a possible scenario where we're testing or majorly updating our configure interface.

For example, if we want to test migrating to a credential store of some sort over file based configuration. In that case, I believe we'd want the flexibility for the configure option to interact with a prerelease image!

Reply:
Hmm, currently `help` for the configure command shows
```console
gh actions-importer configure --help
Options:
  -?, -h, --help  Show help and usage information
```

So, it already doesn't contain options that _most_ other commands do. I feel adding it suggests to the user that `--prerelease` does something for the configure command when it actually doesn't, and could cause confusion that way. Could go either way, I suppose, but It doesn't feel quite right to me to create an option for a command that doesn't actually use it.

Reply:
Ah I didn't realize that `gh-actions-importer` completely handled the configuration logic. I thought at least a portion of it was forwarded on to `valet`.

I hadn't looked into the `ConfigurationService.cs` much. https://github.com/github/gh-actions-importer/blob/25a827612db8d0f9ef2289d7bb9dfb3183c4f730/src/ActionsImporter/Services/ConfigurationService.cs#L7-L29

I see that it's doing all the work to configure the app itself. Makes sense with that context, I'll remove it from the `configure` option.